packages/devtools_app/lib/src/debugger/flutter/console.dart

```diff
@@ -98,17 +97,16 @@ class Console extends StatelessWidget {
     return Stack(
       children: [
         _ConsoleOutput(lines: lines),
-        if (controls.isNotEmpty) ...[
-          _ConsoleControls(
-            controls: controls,
-          ),
-        ],
+        _ConsoleControls(
+          controls: controls,
+        ),
```

Comment:
I'd keep the `isNotEmpty` check here, so we don't end up rendering unneeded containers (I'm planning on reusing this console output in the "Logging" screen, and there are cases where we need to render a completely empty Console)

Reply:
Gotcha, I'll add this back.